src/components/UserProfile.js

```diff
@@ -0,0 +1,277 @@
+const usersRouter = require("express").Router();
```

Comment:
This whole file looks like a duplicate of /api/users.js. This does not belong here in the src/components folder.